### ISSUE TITLE
Remove "and" - appearing between Romanized and original script in browse Gallery view 

### ISSUE THREAD

**Issue by kelea99** [pulibrary/dpul#1576]
### Summary
Reported by Partners in Pakistan for film posters initiative. 

Current state: https://dpul.princeton.edu/pakistani-film/browse/all-exhibit-items (Gallery view, exhibiting "and")
List view, which reads fine: https://dpul.princeton.edu/pakistani-film/catalog?utf8=%E2%9C%93&exhibit_id=pakistani-film&search_field=all_fields&q=

What catalog does (preferred): for list see #12 and #13: https://catalog.princeton.edu/?f%5Blanguage_facet%5D%5B%5D=Arabic&q=&search_field=all_fields

Individual:  https://catalog.princeton.edu/catalog/99115149053506421

### Impact 
_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
@ellen-aa , can you confirm the soft launch date? 
### Priority recommendation

- [ ] asap
- [x] within the next 3 weeks
- [ ] PO will prioritize

## Sudden Priority Justification
_Required if "asap" or "within the next 3 weeks" is checked.  Add "Sudden Priority" and "Maintenance/Research labels_

**Comment by ellen-aa** [comment]
Hi all, did not realize that I needed to respond regarding a soft launch date! I would like to launch as soon as possible. Aside from the removing "and", I can soft launch once we also figure out why "Alternative Title" is not displaying and not currently searchable from the DPUL site - this is critical because we included metadata in this field that greatly enhances overall searchability for the collection. The Alternative Title field contains the Anglicized title on the poster itself, e.g. Prosi: https://dpul.princeton.edu/pakistani-film/catalog/2e5d9c37-9fef-4657-b2cf-c6604a06a4a1

**Comment by kelea99** [comment]
@ellen-aa just pinged you.  The alt titles show. but I found out what the problem is I think! I was searching alt titles and saw them fine....but only items with only one alt title! I looked in figgy and found the items with more than one alt title? do not show in DPUL at all! in the edit mode of the item in dpul, even the alt title field looks blank.

**Comment by kelea99** [comment]
closed ticket for same thing in different view: https://github.com/pulibrary/dpul/issues/1273

**Comment by kelea99** [comment]
User feedback: This was noticed immediately by a user and they asked "how to turn the 'and' off".  perhaps we could get some user input on a "/" versus and "and" between scripts. Perhaps @joycebcat or @ellen-aa ?

**Comment by ellen-aa** [comment]
@kelea99 Yes, very much prefer "/" to "and" for the display in Gallery view. Relatedly, in List view, "and" displays at the front of the titles (https://dpul.princeton.edu/pakistani-film/browse/all-exhibit-items?view=list) - Is it the case that if we replace and with a backslash, the backslash would display at the front of the titles in List view?

**Comment by joycebcat** [comment]
I've complained about the "and" myself.  Definitely an alternative like the "/" is much better but it won't actually  help much if it isn't positioned between the two titles rather than at the beginning. --Joyce

**Comment by kelea99** [comment]
@hackartisan , moving this up to a sudden priority, as there was another request and ask about when this view might be fixed and we have feedback for what is expected. @ellen-aa is also considering holding off on the hard launch until the and is removed. ill edit the ticket accordingly. thanks !

**Comment by hackartisan** [comment]
I don't see "and" in the gallery view. is this fixed? 

Or are we talking about the "and" in the list view between multiple alternative titles? If so, does that have the same priority? And what should it be instead? A slash, or something else, like a newline or a comma?

Thank you!

**Comment by kelea99** [comment]
@hackartisan I see "and" in the gallery view and the "and" in the list view between original script and translit script.  Yes, a slash. we got confirmation from Joyce, Deborah, and Ellen and they were kind enough to leave comments in the thread for us! Let me make a demo of what I see and Ill post here.

**Comment by kelea99** [comment]
https://github.com/user-attachments/assets/50045b04-3fae-4c11-93dc-1502429fd531
